File: SourceCode/Debug Helpers/open.fda.gov/ApiDalc/Enumerations/ReactionOutcome.vb
Imports System.ComponentModel

Namespace Enumerations

    ''' <summary>
    ''' Reaction Outcomes
    ''' </summary>
    ''' <remarks></remarks>
    Public Enum ReactionOutcome

        <Description("Recovered/resolved")>
        RecoveredResolved = 1
        <Description("Recovering/resolving")>
        RecoveringResolving = 2
        <Description("Not recovered/not resolved")>
        NotRecoveredNotResolved = 3
        <Description("Recovered/resolved with sequelae")>
        RecoveredResolvedWithSequelae = 4
        <Description("Fatal")>
        Fatal = 5
        <Description("Unknown")>
        Unknown = 6

    End Enum

End Namespace

File: SourceCode/AspCoreVersion/src/ShopAware.Core/Enumerations/ReactionOutcome.cs
namespace ShopAware.Core
{
    namespace Enumerations
    {
        /// <summary>
        ///     Reaction Outcomes
        /// </summary>
        /// <remarks></remarks>
        public enum ReactionOutcome
        {
            RecoveredResolved = 1,
            RecoveringResolving = 2,
            NotRecoveredNotResolved = 3,
            RecoveredResolvedWithSequelae = 4,
            Fatal = 5,
            Unknown = 6
        }
    }
}